src/Rxjs.res

```diff
@@ -128,7 +128,7 @@ external toObservable: t<'c, 's, 'a> => t<foreign, void, 'a> = "%identity"
 let const: ('b, . t<'c, 's, 'a>) => t<'c, 's, 'b> = (b) => map(. (_, _) => b)
 
 
-@module("rxjs") external mergeArray: array<t<'ca, 'sa, 'a>> => t<foreign, void, 'a> = "merge"
+@module("rxjs") @variadic external mergeArray: array<t<'ca, 'sa, 'a>> => t<foreign, void, 'a> = "merge"
 @module("rxjs") external merge2: (t<'ca, 'sa, 'a>, t<'cb, 'sb, 'a>) => t<foreign, void, 'a> = "merge"
```

Comment:
Ah I see. reading Reacript docs : looks like `@variadic` is for binding res with JS functions that take an arbitrary amount of arguments. ReScript supports modeling those, under the condition that the arbitrary arguments part is homogenous (aka of the same type).

Reply:
@AlexMouton Is this a fix that helps us get further along in FieldTripv1 integration or is this a general RxJS enhancement? Mind just writing your reasoning on the update in the comment? Our plan is to merge this in, update our main repo with the new package version, and then run smokes /nightlies against the upgrade. If everything works as expected, we'll deploy this fix into our main production application. 

At that point, where would we stand on FieldTripv1 issues? Does this fix unblock getting to the bottom of those? Just hoping to get a coordinated road map to help further this along. Cheers!